commit 59f163e86575a98942b01cff0e01b9ed248a6642
Author: Valentin David <me@valentindavid.com>
Date:   Wed Mar 20 22:33:54 2024 +0100

    linux: Disable PCIEASPM_POWER_SUPERSAVE
    
    That does not work on my computer. Some PCIe devices are not
    available. There is probably some power management to do. But when the
    nvme boot disk does not appear during boot, there is not much to
    do. It is better to switch back to BIOS configuration (which is
    PCIEASPM_DEFAULT and enabled by default when PCIEASPM_POWER_* are not
    enabled).

diff --git a/files/linux/fdsdk-config.sh b/files/linux/fdsdk-config.sh
index 736dcd128..b35eb6313 100644
--- a/files/linux/fdsdk-config.sh
+++ b/files/linux/fdsdk-config.sh
@@ -1616,7 +1616,6 @@ if has HAVE_PCI; then
         enable PCIEAER
         module PCIEAER_INJECT
         enable PCIEASPM
-        enable PCIEASPM_POWER_SUPERSAVE
         enable PCIE_ECRC
         enable PCIE_DPC
         if has ACPI; then
